2_SERIES_UPGRADE.md

```diff
@@ -22,6 +22,14 @@ simply change this too:
 import quandl as Quandl
 ```
 
+Additionally if you were relying on a saved version of your `api key` _(authtoken)_ in your scripts please note that this functionality has been dropped from the existing package. To continue to load your api key from the filesystem we recommend you manually load the key and set it via the api config `quandl.ApiConfig.api_key`. Some example code that will do this is:
```

Comment:
Will try to improve the documentation. However if they do not know where the file is they probably will not have used this feature so will indicate safe to ignore.